.travis/deploy_spark_enrich.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I think yes please - it's as good a release as any to rationalise the approach. Thanks @BenFradet